src/playground/src/ui/screen/button.component.tsx

```diff
@@ -45,39 +46,79 @@ class Button extends React.Component<Props, State> {
               status={STATUS}
               size='giant'
               alignment={ALIGNMENT}
-              icon={ICON}
+              icon={(width: number, height: number, color: string) => (
```

Comment:
refactor to component `renderIcon` variable

Reply:
Let's ask @malashkevich.